x/ccv/consumer/types/genesis.go

```diff
@@ -104,9 +109,6 @@ func (gs GenesisState) Validate() error {
 		if gs.ProviderClientId == "" {
 			return sdkerrors.Wrap(ccv.ErrInvalidGenesis, "provider client id must be set for a restarting consumer genesis state")
 		}
-		if gs.ProviderChannelId == "" {
-			return sdkerrors.Wrap(ccv.ErrInvalidGenesis, "provider channel id must be set for a restarting consumer genesis state")
-		}
 		if gs.ProviderClientState != nil || gs.ProviderConsensusState != nil {
```

Comment:
If this is a restart of a running chain where the handshake did not complete, then how is it ever going to get restarted?